src/azure-openapi-validator/tests/basic-test.ts

```diff
@@ -0,0 +1,5 @@
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
```

Comment:
For this file, given the one unit test no, but moving forward these are handy tools